handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -362,51 +132,6 @@ protected void replaceHandler(ChannelHandlerContext ctx, String hostname, SslCon
         }
```

Comment:
@rkapsi no as the `SslContext` should not be released in this case as it is reused later on again. We should only release the engine which we do in `replaceHandler` itself. So all good.